cypress/support/commands.js

```diff
@@ -23,3 +23,7 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+  cy.get(`[placeholder = "${placeholder}"]`);
+})
```

Comment:
It's recommended to have a newline at the end of the file. This can help prevent unexpected behavior when concatenating files or when working with some version control systems 📝

Reply:
ok